examples/vanilla-pusher-functions/README.md

```diff
@@ -1,19 +1,9 @@
-# Realtime Pusher Whiteboard
+# Realtime Pusher Example
 
-This directory is a realtime serverless whiteboard powered by Pusher. The frontend is static vanilla HTML, CSS, and JavaScript and the backend is a single Node.js function.
+This directory is a realtime serverless whiteboard powered by Pusher. The frontend is static vanilla HTML, CSS, and JavaScript and the backend is a single Node.js function both deployed with Vercel and zero configuration.
 
-## Creating This Example
+## Deploy Your Own
 
-To get started with a realtime whiteboard on Vercel, you can use [Vercel CLI](https://vercel.com/download) to initialize the project:
+Deploy your own Pusher project with Vercel.
 
-```shell
-$ vercel init vanilla-pusher-functions
-```
-
-### Deploying From Your Terminal
-
-You can deploy your new project with a single command from your terminal using [Vercel CLI](https://vercel.com/download):
-
-```shell
-$ vercel
-```
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Fedit%2Fmain%2Fexamples%2Fvanilla-pusher-functions&env=APP_ID,KEY,SECRET,CLUSTER&envDescription=Pusher%20Channels&envLink=https%3A%2F%2Fpusher.com%2Fchannels)
```

Comment:
This deploy button doesn't work well 🤔

Reply:
```suggestion
[![Deploy with Vercel](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Ftree%2Fmain%2Fexamples%2Fvanilla-pusher-functions&env=APP_ID%2CKEY%2CSECRET%2CCLUSTER&envDescription=Pusher+Channels&envLink=https%3A%2F%2Fpusher.com%2Fchannels)
```

Reply:
@lucleray This is generated from https://vercel.com/docs/more/deploy-button

Reply:
Oh I see the problem, I used the edit button. Let me update

Reply:
@lucleray Take a look again 

![image](https://user-images.githubusercontent.com/229881/117020704-b084e900-acc4-11eb-84f1-b5012fc14057.png)